src/test/Makefile

```diff
@@ -438,11 +441,24 @@ COMMON_FLAGS = \
 	-O0 \
 	-DUNIT_TEST \
 	-isystem $(GTEST_DIR)/inc \
-	-MMD -MP
+	-MMD -MP \
+	-Wno-c99-extensions \
+	-Wno-reorder
 
 CC_VERSION = $(shell $(CC) -dumpversion)
 CXX_VERSION = $(shell $(CXX) -dumpversion)
 
+# Betaflight supports clang 6 - 10. Please revisit when new version arrive.
+ifeq ($(shell $(CC) -v 2>&1 | grep -c "clang version"), 1)
+    CC_SUPPORTED_VERSION := $(shell expr `clang -dumpversion | cut -f1 -d.` \>= 11)
+    CC_SUPPORTED_VERSION := $(shell expr `clang -dumpversion | cut -f1 -d.` \<= 5)
+else
+    CC_SUPPORTED_VERSION := 0
+endif
+ifeq ($(CC_SUPPORTED_VERSION), 1)
```

Comment:
The naming of this variable is counter-intuitive. It should rather be `CC_UNSUPPORTED_VERSION`.

Reply:
I know this was an unwanted state. Had to tank some energy.